test/datasets/test_iwslt2016.py

```diff
@@ -63,7 +127,11 @@ def tearDownClass(cls):
         cls.patcher.stop()
         super().tearDownClass()
 
-    @parameterized.expand([("train", "de", "en"), ("valid", "de", "en")])
+    @parameterized.expand([
```

Comment:
TODO: can we (should we?) parameterize by each supported lang pair?

Reply:
For complete code coverage I think it would make sense to parameterize by each supported lang pair. I believe we could use the [`nested_params` fn ](https://github.com/pytorch/text/blob/main/test/common/parameterized_utils.py#L26)to generate the cartesian product of language pairs. The main issue I forsee with using that function is it would generate lang pairs that map to the same language (i.e `en ->en`) which I'm not sure would make sense for this dataset.

Reply:
@parmeet also wanted to get your thoughts on this

Reply:
Yes, I would agree. Simulating the exact structure as the original dataset with all the possible use-cases would be the key for coverage.